### PR TITLE
setup DB, add migrations, change main router

### PR DESCRIPTION
**Pull request by vladyslav-lapshyn**
Added migration first 5 tables
Change main Router `/phones` to `/products`
Connect server to DB

### CODE REVIEW

File: src/models/Phone.model.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  @AllowNull(false)
+  @ForeignKey(() => Namespace)
+  @Column({
+    type: DataType.INTEGER,
+    field: 'namespace_id',
+  })
+  namespaceId: number;
+
+  @AllowNull(false)
```
Comment (AlexKherson) [inline src/models/Phone.model.ts:68]: add unique to productId

File: migrations/20230801103934-add-phones-table.cjs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      TABLE_NAME,
+      {
+        id: {
+          type: Sequelize.STRING,
+          primaryKey: true,
+          allowNull: false,
+          unique: true,
```
Comment (AlexKherson) [inline migrations/20230801103934-add-phones-table.cjs:14]: remove unique

File: src/models/Phone.model.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+export class Phone extends Model {
+  @AllowNull(false)
+  @Column({
+    type: DataType.DATE,
+  })
+  createdAt: Date;
+
+  @AllowNull(false)
+  @PrimaryKey
+  @Unique
```
Comment (AlexKherson) [inline src/models/Phone.model.ts:30]: remove unique

File: src/models/Product.model.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    onDelete: 'RESTRICT',
+    foreignKey: 'colorId',
+    targetKey: 'id',
+  })
+  color: Color | null;
+
+  @HasOne(() => Phone, {
+    onDelete: 'CASCADE',
+  })
+  products: Phone | null;
```
Comment (AlexKherson) [inline src/models/Product.model.ts:107]: field item

**Review by ruslanpashkov — APPROVED** [review]

**Pull request merged**